src/views/AutogenView.vue

```diff
@@ -653,16 +653,9 @@ export default {
           return
         }
 
-        if ([401].includes(error.response.status)) {
```

Comment:
@utchoang why not dispatch logout on 401/unauthorise error?

Reply:
@rhtyd I don't have changed it. Maybe it's because I haven't merged with the latest master yet.